tests/integration/components/paper-form-test.js

```diff
@@ -182,6 +185,22 @@ test('form submit button is of type submit', function(assert) {
   assert.equal(this.$('button').attr('type'), 'submit');
 });
 
+test('form submit button component can be customized by passing `submitButtonComponent`', function(assert) {
+  assert.expect(1);
+
+  this.register('component:custom-submit-button', Component.extend({
+    classNames: ['custom-submit-button']
+  }));
+
+  this.render(hbs`
+    {{#paper-form submitButtonComponent='custom-submit-button' as |form|}}
```

Comment:
Can we change to double quotes in hbs code to keep the consistency with other tests/templates?